bearRouter.js

```diff
@@ -0,0 +1,118 @@
+const router = require("express").Router();
+const knex = require("knex");
+
+const knexConfig = {
+  client: "sqlite3",
+  useNullAsDefault: true,
+  connection: {
+    filename: "./data/lambda.sqlite3"
+  }
+};
+
+const db = knex(knexConfig);
+
+router.get("/", async (req, res) => {
+  try {
+    const bears = await db("bears");
+    if (bears) {
+      res.status(200).json(bears);
+    }
```

Comment:
what is your conditional really checking, here? if you want to avoid sending back an empty array (if all items get deleted, for instance) make sure to account for the empty array case and send back a custom message explaining that's what happened--otherwise you don't really need this conditional at all